enso-org/enso
# Update dashboard to design v122 (Part 1)

### Pull Request Description
- Close #9886
  - Update style of buttons in Drive Bar
  - Move "Home" page into a "Start" dialog
  - Remove icons that are no longer needed
  - Remove Backend Switcher in top bar - they have now been converted to categories

- Incidental changes
  - Refactor Backend provider so that both Remote and Local backends are available.
    - This was done because both Cloud and Local backends are now easily accessible by switching tabs - the Local backend no longer has its own views with the hidden category switcher

### Important Notes
None

### Code review tips
- There are a lot of changes in Settings components, for example (and in general, any component that needs to use a backend). However, most of the changes are identical, so only the approach needs to be reviewed - although the settings pages and the other pages should probably be reviewed separately.

### Testing instructions
- Make sure new/changed features work, as mentioned above (drive bar, start page, drag-n-drop)
- Because of the backend changes, all flows in general should be checked, at least at a basic level:
  - Sign up/sign in
  - Offline mode (bottom link on login screen on desktop app)
  - Checking that all settings pages work, both when logged in and not logged in

### Screencasts

https://github.com/enso-org/enso/assets/4046547/d8f385e2-3355-404f-a346-035917d40cf9

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
